components/team_settings_modal/team_settings_modal.tsx

```diff
@@ -13,7 +13,8 @@ const SettingsSidebar = React.lazy(() => import('components/settings_sidebar'));
 import TeamSettings from 'components/team_settings';
 
 type Props = {
-    onHide: () => void
+    onHide: () => void,
+    isCloud: boolean,
```

Comment:
What about using `isCloud?: boolean` so we don't need to pass the flag always and just when we want?